### ISSUE TITLE
Large molecule has final charge

### ISSUE THREAD

**Issue by BJWiley233** [TinkerTools/poltype2#21]
Hey Brandon,

I was running poltype for 90 atom molecule and the `esp` from Psi4 keep getting killed with the basis used in poltype, so need to use less memory and used the `6-311G(d)` basis set.  My final molecule after running analyze had `-0.43844` charge.  I tarred up the folder attached.  It also has negative charge on the `combined.xyz`  of -0.00007 Electrons but not on the original tinker 408956424.xyz file. https://drive.google.com/file/d/13lahKVNAjrJ-wd6aa-nW0qYPdWPKCjIN/view?usp=sharing

```
$ analyze final.xyz -k final.key em | grep -A11 Charge
 Total Electric Charge :                  0.43844 Electrons

 Dipole Moment Magnitude :              17370.525 Debye

```
$ analyze 408956424.xyz -k 408956424.key em | grep -A11 Charge
 Total Electric Charge :                  0.00000 Electrons

 Dipole Moment Magnitude :                  5.629 Debye
```

**Comment by BJWiley233** [comment]
Also is there a good way to look at the CSV files from the final simulations in Excel?  The Greek letters look wonky for me.

![image](https://user-images.githubusercontent.com/26236052/210033962-784543be-dd74-4cff-a56c-ecaa3bcea92e.png)

**Comment by pren** [comment]
Does every atom have multiples assigned. Use "analyze" with ep option.

Thanks,
Pengyu
________________________________

**Comment by lukeanewman** [comment]
Yes, analyze/minimize/dynamic do not complain about missing parameters. I ran smaller molecule fragments on poltype and combined the parameters manually because the molecule was large and the fragmenter protocol was not working. I believe the partial charge is a result of my manual combination but I can not figure out a way to pinpoint my error.
